### PR TITLE
Add doc for SNAT and update offload settings to be used

### PR DESCRIPTION
**Pull request by bpradipt**

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/262#" title="Author self-approved">bpradipt</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [bpradipt]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: modules/5_install/install.tf
Comment (yussufsh) [inline modules/5_install/install.tf:244]: Is it not required for `PUBLIC_INTERFACE`? If not can we remove the line at 237? https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/6746cc11ce9558949b9bc1d9e60ebbeae745b972/modules/5_install/install.tf#L237

**Comment by yussufsh** [comment]
@bpradipt Do we still need the experimental comments or is it still experimental you think?

File: modules/5_install/install.tf
Comment (bpradipt) [reply modules/5_install/install.tf:244]: Yes, we can remove it

**Comment by bpradipt** [comment]
> @bpradipt Do we still need the experimental comments or is it still experimental you think?

let the comment be there

**Review by yussufsh — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/262#" title="Author self-approved">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/262#pullrequestreview-632273660" title="LGTM">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/262#" title="Author self-approved">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/262#pullrequestreview-633249835" title="LGTM">sudeeshjohn</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/262#pullrequestreview-632273660" title="LGTM">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
@yussufsh: new pull request created: #264

<details>

In response to [this](https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/262#issuecomment-818535333):

>/cherrypick release-4.7


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by ltccci** [comment]
@yussufsh: new pull request created: #265

<details>

In response to [this](https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/262#issuecomment-818536217):

>/cherrypick release-4.6


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
